packages-next/fields/src/types/json/index.ts

```diff
@@ -0,0 +1,26 @@
+import type { FieldType, BaseGeneratedListTypes, FieldDefaultValue } from '@keystone-next/types';
+import { resolveView } from '../../resolve-view';
+import type { FieldConfig } from '../../interfaces';
+import { Json, PrismaJsonInterface } from './Implementation';
+
+export type JsonFieldConfig<TGeneratedListTypes extends BaseGeneratedListTypes> =
+  FieldConfig<TGeneratedListTypes> & {
+    defaultValue?: FieldDefaultValue<string>;
+    isRequired?: boolean;
+    ui?: {
+      displayMode?: 'textarea';
+    };
```

Comment:
Let's not have this since you can't change it, if we can let you customise it in the future, we can add it then